PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+# [DEV-]()
+
+## Changes summary
+...
+
+## Risks
+...
+
+## Manual testing done
+...
```

Comment:
Can we drop the `...` too, it's just something to delete otherwise

Reply:
dropped, not sure if should keep the spacing or not? left spacing as is, i.e. one line space, then a line for content, then a line after it before next heading.

Reply:
Think it makes sense right?

Reply:
space is good, follows best practices